atlasdb-config/src/main/java/com/palantir/atlasdb/factory/DefaultLockAndTimestampServiceFactory.java

```diff
@@ -383,6 +385,45 @@ private static LockAndTimestampServices getLockAndTimestampServices(
                 .build();
     }
 
+    private static RemoteTimelockServiceAdapter createRemoteTimelockServiceAdapter(
```

Comment:
nit: this is not consistent with the other methods here that use get, but realistically let's just rename the others to create